### PR TITLE
Added secure server flag for https selenium servers

### PR DESCRIPTION
**Pull request by ray-pankaj**
The server url was resolved by concatenation in the `fromHostAndPort` method in [URL.php](https://github.com/giorgiosironi/phpunit-selenium/blob/master/PHPUnit/Extensions/Selenium2TestCase/URL.php). There was no way to create an https server url from host and port. Added a `secure` flag in the `parameters` which is passed to the method, and the URL is created accordingly.

### CODE REVIEW

**Comment by giorgiosironi** [comment]
Haven't seen this use case before, but it's valid. To minimize the surface of the API, I'd prefer for `$host` to accept values like `https://...` but then it wouldn't be called `host` anymore so that wouldn't be a good direction.

File: PHPUnit/Extensions/Selenium2TestCase/URL.php [hunk 1]
```diff
@@ -71,10 +71,14 @@ public function __construct($value)
     /**
      * @param string $host
      * @param int port
+     * @param bool secure
      * @return PHPUnit_Extensions_Selenium2TestCase_URL
      */
-    public static function fromHostAndPort($host, $port)
+    public static function fromHostAndPort($host, $port, $secure)
     {
+        if($secure) {
```
Comment (giorgiosironi) [inline PHPUnit/Extensions/Selenium2TestCase/URL.php]: coding standards would require `if (` here

**Comment by ray-pankaj** [comment]
@giorgiosironi to keep the changes to a minimum I opted to add a parameter, instead of changing the `host` variable to be the `url`.
Fixed the code formatting error.

**Comment by giorgiosironi** [comment]
The test failure on coverage is not related so this can go in, sorry for the wait but I don't have much time to dedicate to this open source project.

**Pull request merged**